### ISSUE TITLE
[Feature request] Notification aboiut low battery.

### ISSUE THREAD

**Issue by kuba3351** [nwg-piotr/nwg-shell#71]
Hello

I am using nwg-shell in my everyday work on Fedora because it is very nice, but lack a very important feature.

If I disconnect my laptop from a power source, the battery goes down, and laptop is turning off instantly without any warning. Nwg-shell is trying to provide a DE-like experience, and every DE warns about low battery before the device power off.

I think nwg-shell should have this feature.

**Comment by nwg-piotr** [comment]
You're right. I used to have an executor for this long time ago, but I was too lazy to implement it in the panel (my current laptop works always connected to the charger). I'll add it in the nearest future.

**Comment by kuba3351** [comment]
Thank you.

The executor is ok for me if it will be inside default config files provided by nwg-shell.
